tests/unit-compatibility-test/fixtures/sonarqube-dce/application-values.yaml

```diff
@@ -453,10 +453,10 @@ spec:
               value: ""
             - name: SONAR_CE_JAVAOPTS
               value: ""
-            - name: SONAR_HELM_CHART_VERSION
-              value: 10.8.0
             - name: SONAR_LOG_JSONOUTPUT
               value: "false"
+            - name: SONAR_HELM_CHART_VERSION
```

Comment:
Because the `SONAR_LOG_JSONOUTPUT` is being inserted a couple of lines before now. But git/diff misunderstands it as if `SONAR_HELM_CHART_VERSION` was being moved around.